src/services/ansibleLint.ts

```diff
@@ -17,10 +17,11 @@ import { parseAllDocuments } from 'yaml';
 import { IAnsibleLintConfig } from '../interfaces/ansibleLintConfig';
 import { fileExists, hasOwnProperty, withInterpreter } from '../utils/misc';
 import { WorkspaceFolderContext } from './workspaceManager';
+
```

Comment:
This change probably doesn't belong in this PR.

Reply:
Okay. Will keep it separate.